chore(docs): migrate cloud docs to .com (#36389)
gatsbyjs/gatsby: docs/docs/how-to/cloud/deploying-to-gatsby-cloud-hosting.md

---
title: "Deploying to Gatsby Cloud Hosting"

> description: "This tutorial will show you how to deploy your site to Gatsby Cloud hosting."

Reviewer: ```suggestion
description: "Learn how to deploy your site to Gatsby Cloud hosting."
```
